modules/api-gateway/main.tf

```diff
@@ -0,0 +1,227 @@
+# API Gateway- API for ORCA cumulus reconciliation
+resource "aws_api_gateway_rest_api" "orca_cumulus_reconciliation_api" {
```

Comment:
Probably want to generalize the api name to `orca_api` instead of `orca_cumulus_reconciliation_api` since we are consolidating to one API.

Reply:
changed to `orca_api`